zipkin-collector/kafka/src/test/java/zipkin2/collector/kafka/KafkaCollectorRule.java

```diff
@@ -31,7 +31,7 @@
 /** This should be used as a {@link ClassRule} as it takes a very long time to start-up. */
 class KafkaCollectorRule extends ExternalResource {
   static final Logger LOGGER = LoggerFactory.getLogger(KafkaCollectorRule.class);
-  static final String IMAGE = "openzipkin/zipkin-kafka";
+  static final String IMAGE = "openzipkin/zipkin-kafka:latest";
```

Comment:
I think @anuraaga probably meant reproducible as in using a specific version like the other ones. I've copied the version used by those which is easy enough to wildcard replace.

```suggestion
  static final String IMAGE = "openzipkin/zipkin-kafka:2.21.5";
```